### PR TITLE
Add sidebars to web/guide pages

### PR DESCRIPTION
**Pull request by OnkarRuikar**

### CODE REVIEW

File: files/en-us/web/guide/api/index.md [hunk 1]
```diff
@@ -8,15 +8,19 @@ tags:
...
+The web includes a wide array of APIs that can be used from JavaScript to build increasingly more powerful and capable applications, running either on the web, locally, or through technology such as [Node.js](https://nodejs.org/), on a server. On this page you'll find a complete list of all the APIs provided by the full web technology stack.
 
 ## Web APIs from A to Z
 
 {{ListGroups}}
 
 ## See also
 
-- [Web API interface reference](/en-US/docs/Web/API) (an index of all of the interfaces comprising all of these APIs)
+- [Web API interface reference](/en-US/docs/Web/API) (an index of all the interfaces comprising all of these APIs)
```
Comment (bsmth) [inline files/en-us/web/guide/api/index.md:23]:
```suggestion
- [Web API interface reference](/en-US/docs/Web/API) (an index of all interfaces comprising these APIs)
```

**Comment by bsmth** [comment]
Thanks @OnkarRuikar , ccing @wbamberg who has been looking at sidebars a lot recently.

**Review by wbamberg — APPROVED** [review]
Thank you @OnkarRuikar !

These are better than nothing, but our top-level organization (and content in many cases) is terribly poor :(.

**Comment by OnkarRuikar** [comment]
@wbamberg we'll get there one change at a time! :)

I wish [QuickLinksWithSubpages](https://github.com/mdn/yari/blob/main/kumascript/macros/QuickLinksWithSubpages.ejs) macro had variable depth setting and collapsible sub trees. It would've saved us from using such long code:
```
<section id="Quick_links">
  {{ListSubpagesForSidebar("/en-US/docs/Web/Guide/Audio_and_video_delivery")}}
  <ol>
    <li class="toggle">
      <details>
        <summary>Guides</summary>
        <ol>
          {{ListSubpagesForSidebar("/en-US/docs/Web/Guide")}}
        </ol>
      </details>
    </li>
  </ol>
</section>
```

**Pull request merged**